students/BrandonHenson/Lesson2/series.py

```diff
@@ -0,0 +1,46 @@
+'''
+Brandon Henson
+4/2/18
+Lesson 2 series.py
+using index and math to make functions and assertions
+'''
+
+def fibonacci(n):
+    """This function prints out the digit in the Fibonacci series that is in the index number defined by the parameter."""
+    fibo_list = [0,1]
+    for i in range(2, n+1):
+        x = fibo_list[i-2] + fibo_list[i-1]
+        fibo_list.append(x)
+    #printed to test if I'm getting the index number
+    #print(fibo_list[n])
+    return fibo_list[n]
+
+def lucas(n):
+    """This function prints out the digit in the lucas series that is in the index number defined by the parameter."""
+    luc_list = [2,1]
+    for i in range(2, n+1):
+        x = luc_list[i-2] + luc_list[i-1]
+        luc_list.append(x)
+    #printed to test if I'm getting the index number
+    #print(luc_list[n])
+    return luc_list[n]
```

Comment:
Nice work! Note that there is an alternative recursive solution to the problem as well